bert_e/workflow/gitwaterflow/queueing.py

```diff
@@ -167,11 +167,15 @@ def close_queued_pull_request(job, pr_id, cascade):
     dst = job.git.dst_branch = branch_factory(repo, pull_request.dst_branch)
     job.git.cascade.finalize(dst)
 
+    target_branches = job.git.cascade.dst_branches
+    if not target_branches and pull_request.dst_branch.startswith('hotfix/'):
```

Comment:
Shouldn't there be a comment describing why hotfix branches are different ? I assume it is because a hotfix change is not backported to other development branches. But this should be clarified in the code I think.